prisma/schema.prisma

```diff
@@ -10,3 +10,54 @@ datasource db {
   url               = env("DATABASE_URL")
   shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
 }
+
+model user {
+  id       Int        @id @default(autoincrement())
+  name     String     @db.VarChar(255)
+  profile  profile?
+  project  project[]
```

Comment:
You'll have some issues down the line when you come to use this, "project" implies singular but it's an array of projects. I recommend renaming this field to `projects`